helm/pinot/values.yaml

```diff
@@ -52,6 +52,9 @@ securityContext: {}
 probes:
   initialDelaySeconds: 60
   periodSeconds: 10
+  failureThreshold: 1
```

Comment:
This default threshold seems to be easier met when there is a system or network glitch.
Shall we make it as 10?

Reply:
I put the value as per k8s default values and end users can set it as per the need. I will change it to better possible values.